eoscompanion/main.py

```diff
@@ -149,17 +149,79 @@ def do_dbus_register(self, connection, object_path):  # pylint: disable=argument
 
     def do_dbus_unregister(self, connection, object_path):  # pylint: disable=arguments-differ
         '''Invoked when we lose a D-Bus connection.'''
-        log('Lost session d-bus connection at {path}'.format(path=object_path))
+        logging.warning('Lost session d-bus connection at %s', object_path)
         return Gio.Application.do_dbus_unregister(self,
                                                   connection,
                                                   object_path)
 
     def do_activate(self):  # pylint: disable=arguments-differ
         '''Invoked when the application is activated.'''
-        log('Activated')
+        logging.info('Activated')
         return Gio.Application.do_activate(self)
 
 
+COMPANION_APP_CONFIG_FILES = [
+    '/var/run/host/etc/eos-companion-app/config.ini',
```

Comment:
`/var/run` is just a symlink to `/run`, so we should use the latter

Reply:
Fixed to /run...